language/tools/move-package/src/compilation/compiled_package.rs

```diff
@@ -575,11 +578,27 @@ impl CompiledPackage {
         } else {
             Flags::empty()
         };
+        // Partition deps_package according whether src is available
+        let (src_deps, bytecode_deps): (Vec<_>, Vec<_>) = deps_package_paths
+            .clone()
+            .into_iter()
+            .partition_map(|(p, b)| if b { Either::Left(p) } else { Either::Right(p) });
+        // If bytecode dependency is not empty, do not allow renaming
+        if !bytecode_deps.is_empty() {
+            if let Some(pkg_name) = resolution_graph.contains_renaming() {
+                anyhow::bail!(
+                    "Found address renaming in package '{}' when \
+                    building Move model -- this is currently not supported",
```

Comment:
nit: This code is not related to Move model, so we should update this error message.

```suggestion
                    building with bytecode dependencies -- this is currently not supported",
```